# Uriel Isaac Alvarez Cardenas

# Problema 1

Supongamos que se desea usar la prueba de la suma de rangos de Wilcoxon, para detectar un corrimiento en la distribucion 2, con base en muestras de tamaño n1 = 6 y n2 = 8.

a. Se debe usar T1 o T1* como el estadistico de prueba?

b. Cual es la region de rechazo para la prueba si \alpha = .05?

c. Cual es la region de rechazo para la prueba si \alpha = .01?

Creo que son necesarios mas datos para contestar el problema

# Problema 2
observaciones de dos muestras aleatorias e independientes, tomadas de las poblaciones 1 y 2, se dan aqui. use la prueba de la suma de rango de wilcoxon para determinar si la poblacion 1 esta corrida a la izquierda a la izquierda de la poblacion 2.

In [ ]:
# Datos de las muestras
muestra1 <- c(1, 3, 2, 3, 5)
muestra2 <- c(4, 7, 6, 8, 6)

# Prueba de suma de rangos de Wilcoxon
wilcox.test(muestra1, muestra2, alternative = "less")

Warning message in wilcox.test.default(muestra1, muestra2, alternative = "less"):
“cannot compute exact p-value with ties”



	Wilcoxon rank sum test with continuity correction

data:  muestra1 and muestra2
W = 1, p-value = 0.01039
alternative hypothesis: true location shift is less than 0




a. Hipótesis nula (H0): No hay diferencia en las ubicaciones de las poblaciones 1 y 2.
   Hipótesis alternativa (H1): La población 1 está desplazada hacia la izquierda de la población 2.

b. Ordenando la muestra combinada de menor a mayor: 1, 2, 3, 3, 4, 5, 6, 6, 7, 8

   Calculando T1, la suma de los rangos para los valores de la muestra 1:
   T1 = rango(1) + rango(3) + rango(2) + rango(3) + rango(5)
   T1 = 1 + 3 + 2 + 3 + 5
   T1 = 14

   Calculando T1*, utilizando la fórmula:
   T1* = n1 * (n1 + n2 + 1) / 2 - T1
   T1* = 5 * (5 + 5 + 1) / 2 - 14
   T1* = 5 * 11 / 2 - 14
   T1* = 27.5 - 14
   T1* = 13.5

c. La región de rechazo para α = 0.05 en la prueba de suma de rangos de Wilcoxon de una cola (menor que) es cuando T1* es menor o igual al valor crítico.

   Utilizando la tabla de valores críticos de Wilcoxon, el valor crítico correspondiente a α = 0.05 (una cola) y n1 = 5, n2 = 5 es aproximadamente 10.

   Entonces, la región de rechazo para α = 0.05 es T1* ≤ 10.

d. El valor p obtenido en la salida del código es 0.01039. Como el valor p es menor que α = 0.05, podemos rechazar la hipótesis nula.

   Por lo tanto, hay suficiente evidencia para indicar que la población 1 está desplazada hacia la izquierda de la población 2.

# Problema 3
Enfermedad de Alzheimer En algunas pruebas de salud en ancianos, un nuevo medicamento ha restaurado su memoria casi como la de jovenes. pronto se probara en pacientes con enfermedad de Alzheimer, esa fatal enfermedad del cerebro que destruye la mente. Segun el Dr. Gary Lynch, de la Universidad de California en Irvine, el medicamento, llamado ampakana CX-516, acelera señales entre 65 y 70 años de edad, los resultados fueron particularmente sorprendentes. Despues de recibir dosis moderadas de este medicamento, las personas de entre 65 y 70 años de edad calificaron casi tan alto como los jovenes. Los datos siguientes son los numeros de silabas sin sentido recordadas despues de 5 minutos, para 10 hombres de poco mas de 20 años de edad y 10 señores de entre 65 y 70 años. Use la prueba de suma de rango de wilcoxon para determinar si las distribucions para el numero de silabas sin snetido recordadas son iguales para estos dos grupos

In [ ]:
# Datos de los grupos
grupo_20s <- c(3, 6, 4, 8, 7, 1, 1, 2, 7, 8)
grupo_65_70s <- c(1, 0, 4, 1, 2, 5, 0, 2, 2, 3)

# Prueba de suma de rangos de Wilcoxon
wilcox.test(grupo_20s, grupo_65_70s)


Warning message in wilcox.test.default(grupo_20s, grupo_65_70s):
“cannot compute exact p-value with ties”



	Wilcoxon rank sum test with continuity correction

data:  grupo_20s and grupo_65_70s
W = 77.5, p-value = 0.03935
alternative hypothesis: true location shift is not equal to 0


El valor p obtenido es 0.03935. Como el valor p es menor que el nivel de significancia comúnmente utilizado de α = 0.05, podemos rechazar la hipótesis nula de que las distribuciones para el número de sílabas sin sentido recordadas son iguales para los dos grupos. Por lo tanto, hay evidencia suficiente para indicar que hay una diferencia significativa en las distribuciones entre los grupos de hombres de poco más de 20 años y los señores de entre 65 y 70 años.

# Problema 4
Suponga que dos grupos mas de 10 hombres cada uno son probados sobre el numero de silabas sin sentido que podian recordar despues de 5 minutos. No obstenate , a los señores de entre 65 y 70 se les da una dosis moderada de ampakina CX-516. ¿los datos dan suficiente evidencia para concluir que este medicamento mejora la memoria en pacientes de entre 65 y 70 años en comparacion con los de poco mas de 20 años? Use un nivel apropiado de $\alpha$.

In [ ]:
# Datos de los grupos
grupo_20s <- c(11, 7, 6, 8, 6, 9, 2, 10, 3, 6)
grupo_65_70s <- c(1, 9, 6, 8, 7, 8, 5, 7, 10, 3)

# Prueba de suma de rangos de Wilcoxon
wilcox.test(grupo_20s, grupo_65_70s, paired = FALSE)


Warning message in wilcox.test.default(grupo_20s, grupo_65_70s, paired = FALSE):
“cannot compute exact p-value with ties”



	Wilcoxon rank sum test with continuity correction

data:  grupo_20s and grupo_65_70s
W = 53, p-value = 0.8489
alternative hypothesis: true location shift is not equal to 0


El valor p obtenido es 0.8489. Como el valor p es mayor que el nivel de significancia comúnmente utilizado de α = 0.05, no hay evidencia suficiente para concluir que este medicamento mejora la memoria en pacientes de entre 65 y 70 años en comparación con los de poco más de 20 años. No se encontró un corrimiento significativo en la distribución de los grupos después de administrar la dosis moderada de ampakina CX-516.

Los datos no proporcionan suficiente evidencia para concluir que el medicamento mejora la memoria en pacientes de entre 65 y 70 años en comparación con los de poco más de 20 años.